### DISCUSSION TITLE
Do we have portal to maintain the configurations for ocelot?

### DISCUSSION THREAD

**Discussion by KenWang007** [ThreeMammals/Ocelot#1823] [Q&A] [answered]
Currently we use the ocelot.json to config the api routes in ocelot, but if you have lot of apis in your application(more than 100), it's hard to maintain the json file in ocelot , so do we have portal to maintain the configuration file in ocelot ? or do we have a plan to build it ?

**Comment by TomPallister** [comment, marked as answer]
@KenWang007 at the moment the only help for this is using [multiple configuration files](https://ocelot.readthedocs.io/en/latest/features/configuration.html#merging-configuration-files). There is also the [administration API](https://ocelot.readthedocs.io/en/latest/features/administration.html) that might be useful to you.

I don't personally intent to build a UI over the configuration at the moment however you could build one yourself and I would be happy to accept a PR. If you do I suggest using the administration concept so it is under that section.

**Comment by KenWang007** [comment]
@TomPallister Thanks for your reply , ocelot is a very awesome project ,i think it is a first opensource project based on .NET Core on .NET platform, so many developers are  interested on it,  thanks for your contribution. Maybe will build a portal to maintain configuration files in the future.

**Comment by TomPallister** [comment]
@KenWang007 thank you for your kind words :) 

I will keep this issue open as I might get time to build the feature in the future :)

**Comment by dbarkwell** [comment]
@KenWang007 you could use this: https://github.com/dbarkwell/Ocelot.ConfigEditor

**Comment by KenWang007** [comment]
@TomPallister Thanks a lot

**Comment by KenWang007** [comment]
@dbarkwell Thanks
